e2e/testcafe-devextreme/tests/dataGrid/tagBox.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import DataGrid from 'devextreme-testcafe-models/dataGrid';
+import url from '../../helpers/getPageUrl';
+import { createWidget } from '../../helpers/createWidget';
+
+fixture.disablePageReloads`Tagbox Columns`.page(
+  url(__dirname, '../container.html'),
+);
+// T1228720
+test('Datagrid tagbox column should not look broken', async (t) => {
```

Comment:
You test the Generic theme which does not have an issue. I think the test shoul be executed in Material and Fluent theme